bellows/uart.py

```diff
@@ -249,9 +249,11 @@ def _unstuff(self, s):
 
 
 @asyncio.coroutine
-def connect(port, application, loop=None):
+def connect(port, application, loop=None, baudrate=None):
     if loop is None:
         loop = asyncio.get_event_loop()
+    if baudrate is None:
```

Comment:
Instead of making the kwarg default to None and doing this, just make it default to 57600.

Reply:
Right.